CONTRIBUTING.md

```diff
@@ -13,6 +13,9 @@ From there, select “Feature request,” and fill out the form, following the p
 added to the product discussion backlog for prioritization. The GitHub issue that is created will be used for
 communication about the request.
 
+### Consider User Privacy!
+If your feature request in any way impacts user privacy, or could leak Personally-Identifiable Information (PII) about the customer or user of hcdiag, it will need to be cleared via Hashicorp legal first.
```

Comment:
Since this repo is for internal and external customers, let's leave out the HashiCorp legal bit.